packages/version-updater/scripts/postVersioning.mjs

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env node
+/*
+ * Copyright 2024 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+
+import { consola } from "consola";
+import { findUpSync } from "find-up";
+import * as fs from "node:fs";
+import * as path from "node:path";
+import { fileURLToPath } from "node:url";
+import * as semver from "semver";
+
+const workspaceDirPath = getWorkspaceDirPath();
+const clientPackageVersion = getClientPackageVersion();
+
+updateConstVariable(
+  path.join(
+    workspaceDirPath,
+    "packages",
+    "client",
+    "src",
+    "createClient.ts",
+  ),
+  "MaxOsdkVersion",
+  clientPackageVersion,
+);
+
+updateConstVariable(
+  path.join(
+    workspaceDirPath,
+    "packages",
+    "generator",
+    "src",
+    "v2.0",
+    "generateMetadata.ts",
+  ),
+  "ExpectedOsdkVersion",
+  clientPackageVersion,
+);
+
+/**
+ * @param {string} filePath
+ * @param {string} variableName
+ * @param {string} value
+ */
+function updateConstVariable(filePath, variableName, value) {
```

Comment:
nit: doesn't look like you're actually using `value` in the function, can probably just use `clientPackageVersion` directly if you want

Reply:
Great eye. Thank you. I intended to.